ui/jetbrains/src/main/java/com/neueda/jetbrains/plugin/graphdb/jetbrains/services/ExecutorServiceImpl.java

```diff
@@ -19,4 +20,17 @@
             }
         });
     }
+
+    @Override
+    public <T> void runInBackground(Callable<T> task, Consumer<T> onSuccess, Consumer<Exception> onFailure, ModalityState modalityState) {
```

Comment:
Let the original method delegate to this new one with ModalityState.defaultModalityState().

Reply:
done